src/ui/pages/developer/index.tsx

```diff
@@ -29,7 +29,7 @@ export function Developer() {
   const cleanConfig = () =>
     pipe(
       projectRepository.projects.get(),
-      task.traverseSeqArray((project) => projectRepository.removeProject(project.value.projectId)),
+      task.traverseSeqArray((project) => projectRepository.removeProject(project)),
```

Comment:
Could ETA reduce here
```suggestion
      task.traverseSeqArray(projectRepository.removeProject),
```

Reply:
"ETA"?

Reply:
From the Greek letter – should've written Eta :) https://wiki.haskell.org/Eta_conversion